src/s2python/s2_control_type.py

```diff
@@ -35,9 +36,32 @@ def handle_instruction(
         self, conn: "S2Connection", msg: S2Message, send_okay: typing.Callable[[], None]
     ) -> None: ...
 
+    # TODO
     @abc.abstractmethod
     def activate(self, conn: "S2Connection") -> None: ...
 
+    # TODO
+    @abc.abstractmethod
+    def deactivate(self, conn: "S2Connection") -> None: ...
```

Comment:
To me, these TODOs come across as signalling stubs in an example RM implementation. If so, that could be easily cleared up with an inline comment. Consider moving that comment into the function's docstring, so IDEs can pick it up on inspection.

```suggestion
    @abc.abstractmethod
    def deactivate(self, conn: "S2Connection") -> None:
        """Overwrite with the actual deactivation logic of your Resource Manager for this particular control type."""
        pass
```

Or are these TODOs about something else?